3.3-bullseye-slim-minimal/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# tags=articulate/articulate-ruby:3.3-bullseye-slim-minimal
```

Comment:
This is our old pattern, follow the 3.2 structure

Reply:
Oh I didn't scroll down far enough. We don't need to maintain any articulate-ruby images anymore. Just articulate/ruby

Reply:
Oh, gotcha!  That's what you were getting at in the ticket..just the 3.3/base and 3.3/node images then. :+1:

Reply:
Go ahead and resolve if this is now corrected. (want to make sure I understood)